client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-meta-data/motion-meta-data.component.html

```diff
@@ -140,9 +139,9 @@ <h4>{{ 'Motion forwarded to' | translate }}</h4>
 <!-- recommendation referencing motions -->
 <div *ngIf="showReferringMotions && motion?.referenced_in_motion_recommendation_extensions?.length > 0">
     <h4>{{ 'Referring motions' | translate }}</h4>
-    <span *ngFor="let motion of motion.referenced_in_motion_recommendation_extensions; let last = last">
+    <span *ngFor="let motion of getReferencingMotions(); let last = last">
```

Comment:
It might not be a good idea to read from a method here that does a sort of an array.

Reply:
Can you give me a code example?

Reply:
@luisa-beerboom Wrote a more detailed description of the issue with that https://github.com/OpenSlides/openslides-client/pull/2227#discussion_r1175034881